lib/plugins/creative.js

```diff
@@ -2,12 +2,14 @@ const assert = require('assert')
 const Vec3 = require('vec3').Vec3
 const { callbackify, sleep } = require('../promise_utils')
 const { once } = require('events')
+const { Physics } = require('prismarine-physics')
 
 module.exports = inject
 
 function inject (bot, { version }) {
   const Item = require('prismarine-item')(version)
-
+  const mcData = require('minecraft-data')(bot.version)
+  const constantGravity = new Physics(mcData).gravity
```

Comment:
Physics is not a constructor but a regular function, new is not needed. This is a bit over-complicated, but I'm fine with it as it should be temporary.

Reply:
Oh oops my bad, is minecraft-data installed by default too btw ? because that'd be a problem if it doesn't come in the package lol,

Reply:
I don't see how it is over-complicated though, 
since you asked me to import prismarine js, the most obvious way was to call the function to get gravity from it, how would you have done this ?

Reply:
It's over complicated to create a new physic engine just to get a constant. But as I said, I'm fine with it. At the moment there isn't a nice way of doing it, we'll need to rework a bit prismarine-physics.